fortigate-autoscale/aws/index.ts

```diff
@@ -1,11 +1,22 @@
+// export the aws CloudFormation response function
+import * as AwsCfnResponse from './aws-cfn-response';
+// export fortigate-autoscale module files
+export * from '../index';
```

Comment:
You can avoid reaching across like this completely by using 'paths' in tsconfig and 'alias' in webpack.config as discussed.

Treat cross-module dependencies as top-level 'node_modules' even if they are local, by adding path/alias settings in tsconfig/webpack.config.

This decouples the modules and allows them to be moved around later.